nservicebus/sqlserver/operations-scripting.md

```diff
@@ -31,6 +31,10 @@ snippet:sqlserver-powershell-nativesend
 
 snippet:sqlserver-nativesend-usage
 
+The `MessageTypeToSend` dictionary value represents the message .NET type, by default the endpoint receiving the message will use that message type to deserialize the incoming message. Depending on the [versioning](/samples/versioning/sample.md) strategy the type can be expressed in 2 different ways:
```

Comment:
> The `MessageTypeToSend` dictionary value represents the message .NET type, by default the endpoint receiving the message will use that message type to deserialize the incoming message. Depending on the [versioning](/samples/versioning/sample.md) strategy the type can be expressed in 2 different ways:

The `MessageTypeToSend` ~~dictionary~~ value is the .NET type of the message. The receiving endpoint will use the type when deserializing incoming message. Depending on the [versioning](/samples/versioning/sample.md) strategy the type can be specified in the following ways: